opm/simulators/timestepping/AdaptiveTimeSteppingEbos.hpp

```diff
@@ -147,6 +151,8 @@ namespace Opm {
                                  "The maximum factor time steps are elongated after a report step");
             EWOMS_REGISTER_PARAM(TypeTag, double, SolverMaxTimeStepInDays,
                                  "The maximum size of a time step in days");
+            EWOMS_REGISTER_PARAM(TypeTag, double, SolverMinTimeStepInDays,
+                                 "The minimum size of a time step in days, before the simulator throw");
```

Comment:
Typo: throws

Reply:
I changed my mind on this, I do not think we should document this (to the users) as "throwing" at all. I think we should rather say something like: "The minimum time step size in days. If a step cannot converge without getting cut below this step size the simulator will stop."

Also, we should make sure that when chopping the step, we set it to max(current step size * factor, minimum step size), and only terminate after actually trying the minimum step size. So instead of step sizes going (with 1 day minimum) for example 3 days -> 1.5 days -> terminate (as 0.75 days < 1 day) we go 3 days -> 1.5 days -> 1 day -> terminate.

Reply:
> The minimum time step size in days.

That will be very inconvenient for LAB units (time unit hours).